### PR TITLE
CU-2f6xr2r iOS tests with fastlane

### PR DESCRIPTION
**Pull request by newmskywalker**

### CODE REVIEW

**Comment by newmskywalker** [comment]
Task linked: [CU-2f6xr2r iOS with KMM CI tests](https://app.clickup.com/t/2f6xr2r)

File: .github/workflows/ios-testing.yml [hunk 1]
```diff
@@ -4,19 +4,9 @@ on:
...
-    - 'gradlew.bat'
-    - 'iosApp/**'
-    - 'local.properties'
-    - 'settings.gradle.kts'
-    - 'shared/build/xcode-frameworks/**'
-    - 'shared/src/commonMain/**'
-    - 'shared/src/iosMain/**'
-    - 'shared/build.gradle.kts'
+    paths-ignore:
+    - 'androidApp/**'
```
Comment (martyu) [inline .github/workflows/ios-testing.yml:8]: I think there was a reason we didn't do this...I forget though.  @andymiller813 ?

**Comment by martyu** [comment]
Can you wait until #30 is merged and make sure this still works?  I re-enable KMM in that.  Right now it's not building the KMM project.

File: iosApp/fastlane/Snapfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Uncomment the lines below you want to change by removing the # in the beginning
+
+# A list of devices you want to take the screenshots from
+devices([
+  "iPhone 8",
```
Comment (martyu) [inline iosApp/fastlane/Snapfile:5]: Maybe we should check SE, since it’s the smallest screen supported?

**Review by martyu — APPROVED** [review]

File: iosApp/fastlane/Snapfile (hunk 2 above)
Comment (newmskywalker) [reply iosApp/fastlane/Snapfile:5]: I'll change that in #34

**Pull request closed without merging**